PocketKit/Sources/PocketKit/Collections/CollectionViewModel.swift

```diff
@@ -315,8 +299,22 @@ private extension CollectionViewModel {
         return snapshot
     }
 
-    /// Builds and sets snapshot when collection stories are found in Core Data
-    func setCollectionSnapshot(_ identifiers: [NSManagedObjectID]) {
+    /// Builds collections, metadata and stories when collection data are found in Core Data
+    func buildCollection(_ identifiers: [NSManagedObjectID]) {
+        guard let id = identifiers.first,
+        let story = source.viewObject(id: id) as? CollectionStory,
+            let collection = story.collection else {
+            return
+        }
+        self.collection = collection
+        buildActions()
```

Comment:
curious on the though process to add `buildActions()` here

Reply:
we don't want to build the ui (and the actions) before the actual collection is available